airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -70,8 +67,8 @@ def __init__(
         self._start_date = self._start_date.date()
         self._end_date = self._end_date.date()
         self._fields = fields
-        self.action_breakdowns = action_breakdowns or self.action_breakdowns
-        self.breakdowns = breakdowns or self.breakdowns
+        self.action_breakdowns = action_breakdowns or self.ALL_ACTION_BREAKDOWNS
```

Comment:
should this also be cleared? (set to `[]`)

This comment seems to imply that it's still an issue https://github.com/airbytehq/oncall/issues/1083#issuecomment-1329630450

Reply:
@pedroslopez 

The main problem here is backward compatibility.
I cannot change `action_breakdowns = []`
because it will break about 20 existing syncs on the prod cloud.

We need to find a good way to distinguish old created sync from new ones for such cases.